frontend-mini-challenge/components/LoggedInPage.tsx

```diff
@@ -15,12 +15,18 @@ import { useEffect } from "react";
 import { useRouter } from "next/router";
 import { clearAuthToken } from "../lib/authTokenStorage";
 import { useCookies } from "react-cookie";
+import { GetServerSidePropsContext } from 'next';
 
 export default function Page({
   children,
   serverSidePropsCorrectlyCalled,
   isLoggedIn,
   loggedInUserData,
+}: {
+  children: React.ReactNode;
+  serverSidePropsCorrectlyCalled: boolean;
+  isLoggedIn: boolean;
+  loggedInUserData: { firstName: string; email: string }
```

Comment:
This gets copy-pasted all over the place now:

```
{ firstName: string; email: string }
```

What if we decide to add a lastName or a cellphoneNumber? Then we'll have to change the code in a million places. 

Make an interface in one file, and then import it as needed.

Remember DRY